<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BattleAIGame</class>
 <widget class="QDialog" name="BattleAIGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BattleShip Computer Game</string>
  </property>
  <widget class="QLabel" name="GameLogo">
   <property name="geometry">
    <rect>
     <x>178</x>
     <y>10</y>
     <width>200</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>BattleShip</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="CustomButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>330</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Custom Board</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RandomButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>300</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Random Board</string>
   </property>
  </widget>
  <widget class="QSplitter" name="Coordinates">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>150</width>
     <height>27</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="XLabel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FireRowText"/>
   <widget class="QLabel" name="YLabel">
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FireColText">
    <property name="enabled">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>221</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="PlayerWin">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <item>
     <widget class="QLabel" name="PlayerBoardLabel">
      <property name="text">
       <string>Your Board</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="PlayerGameBoard"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="FireButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>150</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Fire</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>60</y>
     <width>231</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="EnemyWin">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <item>
     <widget class="QLabel" name="EnemyBoardLabel">
      <property name="text">
       <string>Opponent's Board</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="OpponentGameBoard">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
